extensions/blocks/instagram-gallery/edit.js

```diff
@@ -60,18 +61,31 @@ const InstagramGalleryEdit = props => {
 				access_token: accessToken,
 				count,
 			} ),
-		} ).then( response => {
+		} ).then( ( { external_name: externalName, images: imageList } ) => {
 			setIsLoadingGallery( false );
 
-			if ( isEmpty( response.images ) ) {
+			if ( isEmpty( imageList ) ) {
 				noticeOperations.createErrorNotice(
 					__( 'No images were found in your Instagram account.', 'jetpack' )
 				);
 				return;
 			}
 
-			setAttributes( { instagramUser: response.external_name } );
-			setImages( response.images );
+			setAttributes( { instagramUser: externalName } );
+			if ( imageList.length < count ) {
+				noticeOperations.removeAllNotices();
+				noticeOperations.createNotice( {
+					status: 'warning',
```

Comment:
Wondering if `warning` is excessive here, I think `info` is enough.
All in all it's just an information notice, I wouldn't want the user to needlessly worry about it. 😅

Reply:
Done.